src/Api/Event/Serializing.php

```diff
@@ -17,6 +17,8 @@
  *
  * This event is fired when a serializer is constructing an array of resource
  * attributes for API output.
+ *
+ * @deprecated in beta 15, removed in beta 16
```

Comment:
almost forgot, I need to move the `formatDate` method as well, possibly in AbstractSerializer